service/src/test/java/bio/terra/profile/service/azure/ApplicationServiceUnitTest.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.profile.service.azure;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
+
+import bio.terra.profile.common.BaseUnitTest;
+import bio.terra.profile.service.crl.CrlService;
+import bio.terra.profile.service.profile.exception.DuplicateTagException;
+import com.azure.resourcemanager.resources.models.ResourceGroup;
+import java.util.Map;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+
+public class ApplicationServiceUnitTest extends BaseUnitTest {
+
+  @Test
+  public void addTagToMrg() {
+    var tenantId = UUID.randomUUID();
+    var subId = UUID.randomUUID();
+    var mrgId = "fake_mrg_id";
+    var crlService = mock(CrlService.class);
+    var resourceGroup = mock(ResourceGroup.class);
+    when(crlService.getResourceGroup(eq(tenantId), eq(subId), eq(mrgId))).thenReturn(resourceGroup);
+    var applicationService = new ApplicationService(crlService);
+
+    applicationService.addTagToMrg(tenantId, subId, mrgId, "fake_tag", "fake_value");
+
+    verify(crlService, times(1))
```

Comment:
`times(1)` is the default so it cam be omitted.
```suggestion
    verify(crlService)
```